Formula/tfel.rb

```diff
@@ -0,0 +1,63 @@
+class Tfel < Formula
+  desc "Code generation tool dedicated to material knowledge for numerical mechanics"
+  homepage "https://thelfer.github.io/tfel/web/index.html"
+  url "https://github.com/thelfer/tfel/archive/refs/tags/TFEL-4.0.tar.gz"
+  sha256 "ed3fb2f59c6b8c9896606ef92276f81942433dd5f60d8130ba07c3af80b039e2"
+  license "GPL-1.0-or-later"
+  head "https://github.com/thelfer/tfel.git", using: :git, branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.9" => :build
+  fails_with gcc: "5"
+
+  def install
+    mkdir "build" do
+      # The generation of the documentation
+      args = ["-Denable-reference-doc=ON", "-Denable-website=ON"]
+      # cmake options to interface MFront with various languages
+      args << "-Dlocal-castem-header=ON"
+      args << "-Denable-fortran=OFF"
+      args << "-Denable-python=ON"
+      args << "-Denable-python-bindings=OFF"
+      args << "-Denable-cyrano=ON"
+      args << "-Denable-lsdyna=ON"
+      args << "-Denable-aster=ON"
+      args << "-Denable-abaqus=ON"
+      args << "-Denable-calculix=ON"
+      args << "-Denable-comsol=ON"
+      args << "-Denable-diana-fea=ON"
+      args << "-Denable-ansys=ON"
+      args << "-Denable-europlexus=ON"
+      args << "-Denable-portable-build=ON"
```

Comment:
Add all these extra flags to the `args` array upon declaration, instead of adding them all one by one after definition, please.